src/web/mjs/engine/Settings.mjs

```diff
@@ -65,6 +68,15 @@ export default class Settings {
             input: types.directory,
             value: path.join( docs, 'Mangas' )
         };
+        //HYPOFLEX
+        let electron = require( 'electron' );
+        this.bookmarkDirectory = {
+            label: 'Bookmarks Directory',
+            description: 'The directory where bookmarks file will be stored',
+            input: types.directory,
+            value: path.join( electron.remote.app.getPath( 'userData' ) )
+            // value: path.join( docs )
```

Comment:
This cant be tested well running  in npm. when clicking the browser icon both on this new directory browser as the already existing one for manga's it returns a {Promise Object} in the field, how can i  get around this.

right  now ive been just  switching the value within the code to test.

Reply:
I forgot to fix the bug in dark theme.
Done now.
https://github.com/manga-download/hakuneko/commit/618ecde29a464e6a838fe03fb6c04738317f11d7